app/messages.py

```diff
@@ -135,6 +135,11 @@
 
 # Login errors
 USER_ENTERED_INCORRECT_PASSWORD = {"message": "Current password is incorrect."}
+USER_ENTERED_CURRENT_PASSWORD = {"message": "New password should not be same "
+                                            "with the current password."}
+WRONG_LENGTH_OF_PASSWORD = {"message": "The new_password field has to be "
```

Comment:
Can you check this method `test_user_password_change_request_password_inferior_to_limit` and implement the message like this instead of adding a new one?